<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlightDialog</class>
 <widget class="QDialog" name="FlightDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>441</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Авиарейс</string>
  </property>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Введите данные авиарейса</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>505</width>
     <height>357</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="num_flight_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Номер авиарейса:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="company_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Авиакомпания:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="from_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Аэропорт отправления:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="to_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Аэропорт прибытия:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="date_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Дата отправления:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="time_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Время отправления:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="state_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Статус:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="all_place_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Всего мест:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="free_place_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Свободных мест:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="num_flight_edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="company_edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="from_edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="to_edit"/>
      </item>
      <item>
       <widget class="QDateEdit" name="date_date"/>
      </item>
      <item>
       <widget class="QTimeEdit" name="time_time">
        <property name="displayFormat">
         <string>HH:mm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="state_box">
        <item>
         <property name="text">
          <string>По расписанию</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Регистрация</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Посадка</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Вылетел</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Задерживается</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Отменён</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="all_place_spin">
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="free_place_spin">
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
